index.md

```diff
@@ -12,35 +12,85 @@ First we set up & capture the content, then we render it in the scaffolding belo
 
+[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png)
+{:.screenshot.zoom}
+
+### Designed & tested
+Cockpit has been designed with your goals in mind. We also routinely test Cockpit with usability studies to make it work the way you'd expect. As a result, Cockpit gets easier to use all the time.
+
+### Team-friendly
+Cockpit is useful for seasoned admins and is friendly enough for those new to Linux. We've designed Cockpit to be friendly for whole teams.
+
+### Packages included
+[Installing and running Cockpit]({{ site.baseurl }}/running.html) is simple. It's already included in most of the major distributions.
+
 ---
 
-[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png){:.screenshot.zoom}
-### No interference
+## Integrated
+
+### Plays well with others
 Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
 
+[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png)
+{:.screenshot.zoom}
+
+### Sign in like normal
+By default, Cockpit uses [system credentials]({{ site.baseurl }}/guide/latest/privileges). You don't need to set up any special accounts. Network-wide logins are also supported through [single-sign-on]({{ site.baseurl }}/guide/latest/sso) and other [authentication]({{ site.baseurl }}/guide/latest/authentication) techniques.
+
+### Self-contained
+Cockpit is self-contained and uses existing system resources. You don't have to worry about setting up a webserver just to use Cockpit.
+
+### Uses existing APIs
+Cockpit uses APIs that already exist on the system. It doesn't reinvent subsystems or add a layer of its own tooling.
+
 ---
 
-[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png){:.screenshot.zoom}
+## Get an overview
+
+### Understand at a glance
+Immediately understand the health of your server. Cockpit's overview page shows current statistics and the status of your system.
+
 ### Multi-server
-You can monitor and administer several servers at the same time. Just add it easily and your server will look after its buddies.
+You can monitor and administer [several servers]({{ site.baseurl }}/guide/latest/feature-machines.html) at the same time. Just add it easily and your server will look after its buddies.
+
+[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png)
+{:.screenshot.zoom}
+
+### Troubleshoot
+Fix pesky problems with ease. No longer worry about SELinux security violations — the fix is just a click away.
```

Comment:
IMHO, SELinux is one of the sore spots, where most actual problems are in fact *not* a click away. This is a promise which we cannot really hold, so maybe use an example about networking or virtual machine troubles? Or make the sentence a bit less strong and "this is an universal solution" :) (Which cannot really exist in principle)